### DISCUSSION TITLE
v4 Log File Issue

### DISCUSSION THREAD

**Discussion by Acabar** [PSAppDeployToolkit/PSAppDeployToolkit#1116] [Q&A] [answered]
In doing some preliminary config and testing in v4 and I've noticed the log file generated doesn't have any time stamps in the Date/Time column when viewing in CMtrace. 
Is anyone else seeing similar or did I miss something in the configs?

Example:
![image](https://github.com/user-attachments/assets/b5f218c6-8343-4ad3-b7c9-100ac0430446)

**Comment by mjr4077au** [comment, marked as answer]
This has been addressed in https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/commit/17b4456442f73c3d23e99d880f2aaaeedc376230, which will be in the next point release.
